docs/user_workloads/accessing_virtual_machines.md

```diff
@@ -161,21 +161,24 @@ This allows for dynamic injection of SSH public keys at runtime by updating the
 
 Please note that new Secrets cannot be attached to a running VM: You must restart the VM to attach the new Secret.
 
-> Note: This requires the qemu-guest-agent to be installed within the guest.
->
-> Note: When using qemuGuestAgent propagation,
-> the `/home/$USER/.ssh/authorized_keys` file will be owned by the guest agent.
+> **Requirement:** The qemu-guest-agent must be installed within the guest.
+
+> **Deprecation Notice:** The implementation supporting qemu-guest-agent versions older than 5.2 is deprecated
```

Comment:
```suggestion
!!! note "Deprecation notice" 
    Support for `qemu-guest-agent` versions older than 5.2 is deprecated and will be removed in a future release.
```
https://squidfunk.github.io/mkdocs-material/reference/admonitions/#changing-the-title